src/components/ProjectWork/ConfirmationCheck.js

```diff
@@ -1,198 +1,149 @@
-              confirmationChecks[2],
-              !confirmationChecks[3],
-              confirmationChecks[4],
-              confirmationChecks[5],
-              confirmationChecks[6]
-            ])
-          }
-        />
-        <span className="confirmation-checks-text">
-          I am committed to continuously learn about systemic racism, in addition to the racial inequalities within the
-          Tech and Design industry.
-        </span>
-      </label>
-      <label>
-        <input
-          name="confirm1"
-          type="checkbox"
-          checked={confirmationChecks[4]}
-          disabled={hasConfirmed}
-          onChange={() =>
-            setConfirmationChecks([
-              confirmationChecks[0],
-              confirmationChecks[1],
-              confirmationChecks[2],
-              confirmationChecks[3],
-              !confirmationChecks[4],
-              confirmationChecks[5],
-              confirmationChecks[6]
-            ])
-          }
-        />
-        <span className="confirmation-checks-text">
-          I am committed to offering my skill set(s) pro bono during the entire duration of this project.
-        </span>
-      </label>
-      <label>
-        <input
-          name="confirm1"
-          type="checkbox"
-          checked={confirmationChecks[5]}
-          disabled={hasConfirmed}
-          onChange={() =>
-            setConfirmationChecks([
-              confirmationChecks[0],
-              confirmationChecks[1],
-              confirmationChecks[2],
-              confirmationChecks[3],
-              confirmationChecks[4],
-              !confirmationChecks[5],
-              confirmationChecks[6]
-            ])
-          }
-        />
-        <span className="confirmation-checks-text">
-          I am committed to supporting the intersectionality of anti-racism efforts.
-        </span>
-      </label>
-      <label>
-        <input
-          name="confirm1"
-          type="checkbox"
-          checked={confirmationChecks[6]}
-          disabled={hasConfirmed}
-          onChange={() =>
-            setConfirmationChecks([
-              confirmationChecks[0],
-              confirmationChecks[1],
-              confirmationChecks[2],
-              confirmationChecks[3],
-              confirmationChecks[4],
-              confirmationChecks[5],
-              !confirmationChecks[6]
-            ])
-          }
-        />
-        <span className="confirmation-checks-text">
-          I am committed to ensuring that the entire duration of this project development and communication efforts
-          remain a safe space for all parties involved.
-        </span>
-      </label>
-      <div className="confirmation-btn-container">
-        <button
-          type="button"
-          className="secondary-button btn confirmation-btn"
+      <p>Please read and agree to the following guidelines in order to commit to this project:</p>
+      <div className="checklist">
```

Comment:
I was planning on refactoring this later so you don't need to if you don't want but we can turn all these statements into an array and just map over them to be cleaner